docs/content/unit-testing.fsx

```diff
@@ -87,7 +87,7 @@ module MyTests =
 
     //unit test
     let ``best sales rep in Canada``() = 
-        let expected = Some( GetSalesChampion.Record("Jos�", "Saraiva"))
+        let expected = Some( GetSalesChampion.Record("Jos�", "Saraiva"))
```

Comment:
there is some kind of encoding glitch here.

Reply:
Fixed - replaced with a unicode é.